pkg/network/node/egressip_test.go

```diff
@@ -130,6 +127,54 @@ func assertNoOVSChanges(eip *egressIPWatcher, flows *[]string) error {
 	return assertOVSChanges(eip, flows)
 }
 
+func compareGroups(groups []string, expected map[uint32][]string) bool {
+	returnedGroups := make(map[uint32][]string)
+
+	for _, group := range groups {
+		var groupID uint32
+		// it is difficult to separate the returned groups into a meaningful form
+		splitGroup := strings.SplitN(group, "bucket=", 2)
```

Comment:
there should probably be something in `pkg/network/node/ovs/parse.go` to parse the group lines, like we have with `ParseFlow`

Reply:
done